library/ssl_srv.c

```diff
@@ -1877,6 +1877,7 @@ static int ssl_parse_client_hello( mbedtls_ssl_context *ssl )
     {
         for( j = 0, p = buf + ciph_offset + 2; j < ciph_len; j += 2, p += 2 )
             for( i = 0; ciphersuites[i] != 0; i++ )
+            {
```

Comment:
Ideally please combine the correction commit into the previous commit to give a cleaner history and simplify review. This could be done with e.g.:

git reset --hard origin/development
git cherry-pick 9e037f3b
git cherry-pick -n d423b95b0
git commit --amend
git cherry-pick 30e4eed00ebd

Reply:
Done